course_viewer/course_viewer/models.py

```diff
@@ -0,0 +1,111 @@
+from django.db import models
+
+
+class State(models.Model):
+    '''
+    The Transfer State
+    '''
+    state_code = models.CharField(
+                    max_length=2,
+                    unique=True,
+                    help_text='The abbreviation of the transfer college state')
+
+    state_name = models.CharField(
+                    max_length=200,
+                    unique=True,
+                    help_text='The long name of the transfer college state')
+
+    def __str__(self):
+        return self.state_name
+
+    class Meta:
+        ordering = ['state_code']
+        verbose_name = 'state'
+        verbose_name_plural = 'states'
+        db_table = 'states'
+
+
+class College(models.Model):
+    '''
+    The Transfer College
+    '''
+    college_code = models.CharField(
+                    max_length=15,
+                    primary_key=True,
+                    help_text='The code of the transfer college')
+    college_name = models.CharField(
+                        max_length=50,
+                        help_text='The name of the transfer college')
+
+    def __str__(self):
+        return self.college_name
+
+    class Meta:
+        ordering = ['college_code']
+        verbose_name = 'college'
+        verbose_name_plural = 'colleges'
+        db_table = 'colleges'
+
+
+class Course(models.Model):
+    '''
+    The Transfer Course
+    '''
+    course_name = models.CharField(
+                        max_length=50,
+                        help_text='The name of the transfer course')
+    course_id = models.CharField(
+                        max_length=15,
+                        help_text='The id of the transfer course')
+    course_credit = models.IntegerField(
+                    help_text='The number of credits the transfer course is')
+    course_number = models.CharField(
+                        max_length=15,
+                        help_text='The course number of the transfer course')
+    course_mtu_course_id = models.ForeignKey(
+                'MTUCourse',
+                related_name='course_mtu_course_id',
+                on_delete=models.CASCADE,
+                help_text='The MTU course associated with the transfer course')
+    course_college_code = models.ForeignKey(
+                            'College',
+                            related_name='course_college_code',
+                            on_delete=models.CASCADE,
+                            help_text='The college that the course is at')
+    course_state_code = models.ForeignKey(
+                            'State',
+                            related_name='course_state_code',
+                            on_delete=models.CASCADE,
+                            help_text='The state the course is located in')
```

Comment:
Basically all the variables in this could use some changing on the naming of them.  Without the help text it becomes rather unreadable and confusing without already knowing the output contents.